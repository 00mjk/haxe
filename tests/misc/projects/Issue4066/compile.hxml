-main Main
-cp src1
-cp src2
--interp