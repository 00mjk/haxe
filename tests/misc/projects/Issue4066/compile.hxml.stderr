ok
Never mind, do something else.